swift-sdk/IterableAPI.swift

```diff
@@ -16,23 +16,23 @@ public final class IterableAPI : NSObject {
     /// You should call this method and not call the init method directly.
     /// - parameter apiKey: Iterable API Key.
     public static func initialize(apiKey: String) {
-        implementation = IterableAPIImplementation.initialize(apiKey: apiKey, launchOptions: nil, config:IterableConfig(), dateProvider: SystemDateProvider())
+        implementation = IterableAPIInternal.initialize(apiKey: apiKey)
```

Comment:
Would it make sense to also rename this field from `implementation` to `internal`?